azurefunctions-extensions-bindings-eventhub/azurefunctions/extensions/bindings/eventhub/eventDataConverter.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+
+import collections.abc
+from typing import Any, Optional, get_args, get_origin
+
+from azurefunctions.extensions.base import Datum, InConverter, OutConverter
+from .eventData import EventData
+
+
+class EventDataConverter(
+    InConverter,
+    OutConverter,
+    binding="eventHub",
+    trigger="eventHubTrigger",
+):
+    @classmethod
+    def check_input_type_annotation(cls, pytype: type) -> bool:
+        if pytype is None:
+            return False
+        
+        # The annotation is a class/type (not an object) - not iterable
+        if (isinstance(pytype, type)
+                and issubclass(pytype, EventData)):
+            return True
+        
+        # An iterable who only has one inner type and is a subclass of SdkType
+        return cls.__is_iterable_supported_type(pytype)
+    
+    @classmethod
+    def __is_iterable_supported_type(cls, annotation: type) -> bool:
+        # Check base type from type hint. Ex: List from List[SdkType]
+        base_type = get_origin(annotation)
+        if (base_type is None
+                or not issubclass(base_type, collections.abc.Iterable)):
+            return False
+
+        inner_types = get_args(annotation)
+        if inner_types is None or len(inner_types) != 1:
```

Comment:
In the library we support a list of more than one object. We should support it here too.
https://github.com/Azure/azure-functions-python-library/blob/dev/azure/functions/meta.py#L36